contracts/0.8.25/vaults/Delegation.sol

```diff
@@ -53,23 +53,15 @@ contract Delegation is Dashboard, IReportReceiver {
      */
     bytes32 public constant STAKER_ROLE = keccak256("Vault.Delegation.StakerRole");
 
-    /** 
-     * @notice Role for the operator
-     * Operator can:
+    /**
+     * @notice Role for the node operator
+     * Node operator rewards claimer can:
      * - claim the performance due
      * - vote on performance fee changes
      * - vote on ownership transfer
-     * - set the Key Master role
      */
     bytes32 public constant OPERATOR_ROLE = keccak256("Vault.Delegation.OperatorRole");
 
-    /**
-     * @notice Role for the key master.
-     * Key master can:
-     * - deposit validators to the beacon chain
-     */
-    bytes32 public constant KEY_MASTER_ROLE = keccak256("Vault.Delegation.KeyMasterRole");
```

Comment:
It may be preliminary to get rid of this role.

Reply:
Why? Keymaster does not have any duties in the delegation contract, since they use the base vault for depositing validator keys.

Reply:
It still can be a useful to have a role that can only deposit, but have no access to fees or role setup.